src/app/(auth)/_components/PasswordResetCodeForm.tsx

```
@@ -0,0 +1,124 @@
+                    </InputOTPGroup>
+                  </InputOTP>
+                </FormControl>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
+          <Button
+            type="submit"
+            className='w-full'
+          >
+            {false ? <Spinner size="sm" /> : "Submit"}
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)